scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -107,7 +107,7 @@ object ScaladocParser {
   private def linkParser[_: P]: P[Link] = P {
     def end = space | linkSuffix
     def anchor = P((!end ~ AnyChar).rep(1).!.rep(1, sep = spaces1))
-    def pattern = linkPrefix ~ (anchor ~ linkSuffix ~ punctParser.!)
+    def pattern = linkPrefix ~ (anchor ~ linkSuffix ~ wordParser.?.!)
```

Comment:
do you need `.!` after the `wordParser` (since wordparser is already capturing)? also, `wordParser` returns a `Word`; if you need a string, should it be `labelParser` instead? or if it can be empty (labelParser is not), should it be `(!space ~ AnyChar).rep`?

ended up with lots of questions. last one was probably most relevant.

Reply:
thx! I missed that there is a `labelParser`